src/js/view/components/chapters.mixin.js

```diff
@@ -58,6 +58,7 @@ define([
             var _this = this;
             _.each(this.cues, function (cue) {
                 cue.align(duration);
+                // mousemove/mouseout because this currently mouse specific functionality.
```

Comment:
I added that because I wasn't modifying mouseover/mouseout logic in the touch refactor because we aren't duplicating this functionality across mouse and touch interaction.  I'll remove it since its unnecessary.